fix(DashboardChartSummary): use `null` for `nullable` values

enzo-bitfly opened the pull request:
See: BEDS-1548

marcel-bitfly commented on frontend/components/dashboard/chart/DashboardChartSummary.vue:
```diff
@@ -208,7 +208,10 @@ const loadData = async () => {
         }
 
-        const data = element.data.map(datapoint => datapoint !== undefined ? datapoint * 100 : undefined)
+        const data = element.data.map(datapoint =>
+          (datapoint !== null && datapoint !== undefined)
+            ? datapoint * 100
+            : null)
```
**suggestion:** simplify
```suggestion
        const data = element.data.map(datapoint =>
          (
          datapoint === null || datapoint === undefined)
            ? null
            : datapoint * 100
            )
```

enzo-bitfly replied:
done

remoterami committed:
refactor(api): removed undefined from json response

marcel-bitfly committed:
fix: chart and tooltip for values of `null`

For example for `proposal efficiency` or `sync efficiency` values can also be `null`.
In the chart there should not be a value (instead of `0`) and in tooltip there shouldn't be
`percentage calculation`.

See: BEDS-1548
